src/contradictory_claims/claims/extract_claims.py

```diff
@@ -0,0 +1,102 @@
+"""Function to extract Claims"""
+
+import pandas as pd
+import numpy as np
+import os
+
+from tqdm import tqdm
+import torch
+from torch.nn import ModuleList, Linear
+import torch.nn.functional as F
+from nltk import word_tokenize, sent_tokenize
+
+from allennlp.models.archival import load_archive
+from allennlp.predictors import Predictor
+from allennlp.common.file_utils import cached_path
+from allennlp.common.util import JsonDict
+from allennlp.data import Instance
+from allennlp.modules import Seq2VecEncoder, TimeDistributed, TextFieldEmbedder, ConditionalRandomField, FeedForward
+
+from .utils import read_json
+from .utils import ClaimAnnotationReaderJSON, CrfPubmedRCTReader
+from .utils import DiscourseClassifierPredictor
+from .utils import ClaimCrfPredictor
+
+
+
+def load_model(MODEL_PATH:str = "https://s3-us-west-2.amazonaws.com/pubmed-rct/model_crf.tar.gz", WEIGHT_PATH:str  = "https://s3-us-west-2.amazonaws.com/pubmed-rct/model_crf_tf.th"):
```

Comment:
we might want to disambiguate between this model and the model for contradiction detection. Maybe this one we call `load_claims_extraction_model` or something like that?